lib/impl/AzureCosmosTvm.js

```diff
@@ -77,6 +96,46 @@ class AzureCosmosTvm extends Tvm {
       partitionKey
     }
   }
+
+  async createUserAndPermission (/** @type {cosmos.Database} */ database, userId, permissionDefinition, resourceTokenExpirySeconds) {
```

Comment:
since this function only needs the `id` why not just pass that in?

Reply:
it needs the cosmos db client (database) + userId + permission creation payload (which includes the id) + expiry

Reply:
permissionDefinition.id that is ...